### PR TITLE
Ansible Playbook to run multi image setup as Docker Swarm Services

### LINKED ISSUE #139: Ansible script to deploy multi image setup as Swarm Service
Assume that you localhost has a swarm manager running.

### PR DESCRIPTION
**Pull request by Manas23601**
Fixes #139 
- Pushes Docker Compose Images to Docker Swarm
- Gets Environment keys from Hashicorp Vault
- Docker Images are built from AI-Tools Public Repository Source

### CODE REVIEW

**Comment by Manas23601** [comment]
Sample Inventory File I used to publish it to localhost
```
[swarm_manager]
2405:201:e001:803a:4e63:ae8b:b6ee:b97 ansible_connection=ssh ansible_user=manas ansible_ssh_private_key_file=/home/manas/.ssh/id_rsa
```

**Comment by singhalkarun** [comment]
Hey @Manas23601 ansible playbook should not be dependent on any external file. Since, ai-tools is a public repository, it should download config.json and generate.sh, then generate docker-compose.yml and then start the services on Swarm.

You may assume that all the dependencies related to docker are available in machine.

**Comment by Manas23601** [comment]
in the future we could just pull the images from github packages, We wouldn't need to pull the repository as well

File: swarm.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      environment:
+        VAULT_ADDR: "{{ vault_address }}"
+        VAULT_TOKEN: "{{ vault_token }}"
+
+    - name: Convert Docker Compose to Docker Stack
+      command: docker stack deploy -c ./docker-compose-generated.yaml ansible_stack
+      become_user: "{{ ansible_user }}"
+      environment: './sample_file.env'
+      args:
+        chdir: /home/manas/Codes/ansible
```
Comment (singhalkarun) [inline swarm.yml:43]: Please ensure that script is not dependent on any system level details.

**Comment by Manas23601** [comment]
Currently, we cannot check because the images are not public yet.

**Comment by singhalkarun** [comment]
> Currently, we cannot check because the images are not public yet.

Can you please make relevant changes to the packages workflow to ensure they are public?

**Pull request closed without merging**